zfec/_version.py

```diff
@@ -8,6 +8,8 @@
 # This file is released into the public domain. Generated by
 # versioneer-0.15 (https://github.com/warner/python-versioneer)
 
+from __future__ import print_function
+
```

Comment:
https://docs.python.org/2/library/__future__.html

if i understand that correctly, this will require python >= 2.6.0a2.

so, guess it's time to drop everything < 2.6.

and i'ld also drop 2.6 asap, more and more tools in the python world make troubles on 2.6, so even testing it is sometimes not that easy.